setup.py

```diff
@@ -15,12 +22,14 @@
     url = 'https://bitbucket.org/kmike/yandex-maps/',
 
     description = 'Yandex.Maps API python wrapper with optional django integration.',
-    long_description = open('README.rst').read().decode('utf8') + open('CHANGES.rst').read().decode('utf8'),
+    long_description = open('README.rst').read() + open('CHANGES.rst').read(),
```

Comment:
Hm. How does it fail? I thought that `setup` doesn't care much about the type of `long_description`. Concatenation won't fail because `README` and `CHANGES` are either both UTF or not.

If that's a problem, then maybe we can go with:
```
import codecs
''.join(codecs.open(name, 'r',encoding='utf8') for name in ('README.rst', 'CHANGES.rst')
```